app/javascript/packages/document-capture/components/document-capture.jsx

```diff
@@ -44,18 +44,21 @@ function DocumentCapture({ isLivenessEnabled = true }) {
     isMobile && {
       name: 'intro',
       title: t('doc_auth.headings.document_capture'),
-      component: MobileIntroStep,
+      form: MobileIntroStep,
     },
     {
       name: 'documents',
       title: t('doc_auth.headings.document_capture'),
-      component: DocumentsStep,
+      form: DocumentsStep,
+      footer: isMobile
```

Comment:
I'm curious why we're leaving this message off for mobile devices?

Reply:
> I'm curious why we're leaving this message off for mobile devices?

It's from the design, but I believe the reason is that we show an equivalent text already on the mobile-only introduction screen.

See screenshot at #4030.